pkg/client/didexchange/client.go

```diff
@@ -12,12 +12,13 @@ import (
 	"fmt"
 	"sync"
 
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/wallet"
```

Comment:
import is grouped wrong.

Reply:
looks like I need to fix my goland config